### ISSUE TITLE
Amtsenthebung suggest a relation to the "enthobenes amt" and to a substament. Aren't they synonyms?

### ISSUE THREAD

**Issue by GVogeler** [acdh-oeaw/apis-instance-manmax#47]
![grafik](https://github.com/acdh-oeaw/apis-instance-manmax/assets/324006/2f52e85d-bac5-4666-b6c3-032d345ed0c9)

**Comment by richardhadden** [comment]
No: the role-removed-from is a Role. The substatement is the "Role Occupation" that was ended by the outer statement. The outer is the "event", the inner a "state"...

Not synonymous, but the inner statement is arguably redundant/easily inferrable.
